constructr-akka/src/main/scala/de/heikoseeberger/constructr/akka/Constructr.scala

```diff
@@ -69,9 +70,17 @@ final class Constructr private extends Actor with ActorLogging with ActorSetting
   override def receive = Actor.emptyBehavior
 
   private def active(machine: ActorRef): Receive = {
-    case Terminated(`machine`) =>
-      log.error("Terminating the system, because constructr-machine has terminated!")
-      context.system.terminate()
+    case Terminated(`machine`) => {
+      val cluster = Cluster(context.system)
+      val self = cluster.selfAddress
```

Comment:
Please rename to `selfAddress`, because `self` is already defined and commonly known as the self `ActorRef`.